doc/API.md

```diff
@@ -991,6 +992,27 @@ lists all files in the download queue.
 }
 ```
 
+#### /renter/downloads/clear [POST]
+
+Clears the download history of the renter for a range
+of unix time stamps.  Both parameters are optional, if
+no parameters are provided, the entire download history
+will be cleared.  To clear a single download, provide the
+timestamp for the download as both parameters.  Providing
+only the before parameter will clear all downloads older
+than the timestamp.  Conversely, providing only the after
+parameter will clear all downloads newer than the timestamp.
```

Comment:
is this block formatted correctly? Looks like either the earlier lines cut off too early, or the later lines cut off too late. You can use `gq` in vim to format a block of text to a configured width (default 80 on most distros)